admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -63,13 +63,20 @@ else
     echo "No existing ${CONTAINER_NAME} container on any nodes"
 fi
 
+USE_JEMALLOC='/qserv/stack/stack/current/Linux64/jemalloc/master-g708711ce34/lib/libjemalloc.so'
```

Comment:
These scripts run outside of the eups setup environment, which leaves JEMALLOC_DIR undefined.